src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -1191,6 +1206,10 @@
   "advancements.minecolonies.build.farmer.description": "To start food production for your citizens, build a Farm. You'll need to place the Field (scarecrow) to mark where your Farmer will farm.",
   "advancements.minecolonies.build.citizen.title": "Home Sweet Home",
   "advancements.minecolonies.build.citizen.description": "Your citizens need a place to live! Build them a House - each level houses one citizen.",
+  "advancements.minecolonies.citizen.bury.title": "A sad day indeed",
+  "advancements.minecolonies.citizen.bury.description": "Bury your first citizen, it's a sad day for everyone, but a litle better than letting their grave decay away.",
+  "advancements.minecolonies.citizen.resurrect.title": "A wonderfull day indeed",
+  "advancements.minecolonies.citizen.resurrect.description": "Your gravediggers may be able to resurrect a fallen citizen!",
```

```suggestion
  "advancements.minecolonies.citizen.resurrect.description": "Your Gravedigger resurrected a fallen citizen!",
```
if I'm understanding this right